containers/Dockerfile.buildenv

```diff
@@ -39,7 +39,7 @@ RUN apt-get update -y \
                           libopenblas-dev liblapack-dev \
                           libhdf5-dev hdf5-tools \
                           libgsl0-dev \
-                          clang-6.0 clang-8 clang-9 \
+                          clang-6.0 clang-7 clang-8 clang-9 \
```

Comment:
why not remove clang-6 here?

Reply:
To avoid breaking everyone's PRs :) I'll remove clang-6 in a few week